ynput/OpenPype
# Max: switching versions of maxScene maintain parentage/links with the loaders

## Changelog Description
When using scene inventory to manage or update the version of the loading objects, the linked modifiers or parentage of the objects would be kept.

Meanwhile, loaded objects from all loaders no longer parented to the container with OP Data. 


## Additional info
#5406 affects how this PR works.
It can be ready for review after #5406 merged.
It affects #5434 

## Testing notes:
1. Launch Max via OP
2. Click Load.. via OP Menu
3. Load objects from all loaders
4. Click Manage... via OP Menu
5. Switch Version or Update to the latest by right click
6. Switch the version twice to ensure no bug has been found
